pkg/build/api/types.go

```diff
@@ -476,3 +479,12 @@ type BuildLogOptions struct {
 	// is not available yet. Otherwise the server will wait until the build has started.
 	NoWait bool
 }
+
+// SecretSpec specifies a secret to be included in a build pod and its corresponding mount point
+type SecretSpec struct {
```

Comment:
Why do I need to be able to specify the mount location?  If this is just for custom builds, isn't this really a volume source and a mount spec?  If this is for other builds, how is a user supposed to know where to mount things?